### ISSUE TITLE
Headers in response spec not added as expected

### ISSUE THREAD

**Issue by darrenwhite24** [johanhaleby/stub-http#6]
```
(with-routes! 
  {"/something" {:status       200
                 :content-type "application/json"
                 :headers      {"Some Header" "some value"}}}
  (clj-http.client/get (str uri "/something")))
```
Unlike when content-type is set, the headers that are set aren't available in the response.

```
{:request-time 201,
 :repeatable? false,
 :protocol-version {:name HTTP, :major 1, :minor 1},
 :streaming? true,
 :chunked? true,
 :reason-phrase OK,
 :headers {Content-Type text/html, Connection close, Transfer-Encoding chunked},
 :orig-content-encoding nil,
 :status 200,
 :length -1,
 :body ,
 :trace-redirects []}
```

**Comment by johanhaleby** [comment]
Thanks for noticing this! I've fixed it now.

**Comment by johanhaleby** [comment]
I've also released 0.2.4 which should address this issue.
